src/core/plugins/postmarketos/plugin.spec.js

```diff
@@ -0,0 +1,37 @@
+const api = require("./api.js");
+jest.mock("./api.js");
+api.getImages.mockResolvedValue("a");
+api.getChannels.mockResolvedValue(["1.0", "nightlies"]);
+const asteroid_os = new (require("./plugin.js"))({
+  os: { name: "Asteroid OS" },
```

Comment:
lots of asteroid in this file

Reply:
fwiw I haven't out any effort into linting, or even touched the spec files, as i planned to do that after i got it feature complete. 

thanks for the tips on linting - I didn't know how to do it in node. I'll do a pass and take a look at the tests too, once i get decompression working. it might be best to wait until then to review style/lint etc.